### PR TITLE
Add ActivationSummary screen to end of onboarding

### PR DESCRIPTION
**Pull request by devinjameson**
Why: when I complete onboarding, I would like to be be shown a summary of my activation status so I know whether the app is working or not.

This commit:
- Adds a screen to the end of the activation flow that displays the activation status

Screen Shot 2020-08-24 at 10 18 47 AM
Screen Shot 2020-08-27 at 12 30 03 PM

### CODE REVIEW

File: src/Activation/ActivationSummary.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+  const handleOnPressGoToHome = () => {
+    completeOnboarding()
+  }
+
+  const handleOnPressOpenSettings = async () => {
+    Linking.openSettings()
+    completeOnboarding()
+  }
+
+  const image = isAppSetupComplete
```
Comment (amandabeiner) [inline src/Activation/ActivationSummary.tsx:47]: What do you think about turning these into a configuration object and then using one ternary to return the proper object?

```javascript
const appSetupCompleteContent = {
  headerImage: Images.CheckInCircle,
  headerText: ...,
  bodyText: determineBodyText(isAppSetupComplete, isLocationOffAndNeeded)
}

const screenContent = isAppSetupComplete ? appSetupCompleteContent : appSetupIncompleteContent

<Image source={screenContent.image} />
```

File: src/Activation/ActivationSummary.tsx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+  StyleSheet,
+  Image,
+} from "react-native"
+import { useTranslation } from "react-i18next"
+
+import { usePermissionsContext } from "../PermissionsContext"
+import { useOnboardingContext } from "../OnboardingContext"
+import { useApplicationName } from "../More/useApplicationInfo"
+import { GlobalText } from "../components"
+import { Button } from "../components"
```
Comment (johnschoeman) [inline src/Activation/ActivationSummary.tsx:16]:
```suggestion
import { Button, GlobalText } from "../components"
```

File: src/Activation/ActivationSummary.tsx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  const { exposureNotifications } = usePermissionsContext()
+  const { status } = exposureNotifications
+  const { authorized, enabled } = status
+
+  const isProximityTracingOn = authorized && enabled
+  const isAppSetupComplete =
+    isProximityTracingOn && isBluetoothOn && !isLocationOffAndNeeded
+
+  const handleOnPressGoToHome = () => {
+    completeOnboarding()
```
[the remark this answers is not in the document]
Comment (devinjameson) [reply src/Activation/ActivationSummary.tsx:39]: Yep!

File: src/Activation/ActivationSummary.tsx (hunk 2 above)
Comment (devinjameson) [reply src/Activation/ActivationSummary.tsx]: Yeah I think @amandabeiner were thinking of tackling that tomorrow and building some kind of `ActivationContext` to handle all that stuff, plus the status of Bluetooth and Location.

File: src/Activation/ActivationSummary.tsx (hunk 1 above)
Comment (devinjameson) [reply src/Activation/ActivationSummary.tsx]: Good idea!

File: src/Activation/ActivationSummary.tsx [hunk 4]
```diff
@@ -0,0 +1,174 @@
+import React, { FunctionComponent } from "react"
```
Comment (devinjameson) [inline src/Activation/ActivationSummary.tsx:1]: @johnschoeman Would you be so kind as to give this file a once-over and lmk your thoughts?

**Pull request merged**